Cargo.toml

```diff
@@ -12,14 +12,15 @@ hex = "0.4.3"
 histogram = "0.6.9"
 home = "0.5.3"
 lazy_static = "1.4.0"
+md5 = "0.7.0"
 metrics = "0.20.0"
 metrics-util = "0.14.0"
 parking_lot = "0.12"
 pea2pea = "0.40"
 rand = "0.8"
 rand_chacha = "0.3"
 sha2 = "0.10"
-spectre = "0.4.0"
+spectre = { git = "https://github.com/niklaslong/spectre" }
```

Comment:
Perhaps depend on a specific commit here (i.e. using `rev`) so it doesn't break immediately if we make breaking changes to spectre?